Dockerfile

```diff
@@ -1,42 +1,13 @@
-FROM quay.io/condaforge/mambaforge
+FROM python:3.10.8
+WORKDIR /home/pandas
 
-# if you forked pandas, you can pass in your own GitHub username to use your fork
-# i.e. gh_username=myname
-ARG gh_username=pandas-dev
-ARG pandas_home="/home/pandas"
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get install -y build-essential
 
-# Avoid warnings by switching to noninteractive
-ENV DEBIAN_FRONTEND=noninteractive
+# hdf5 needed for pytables installation
+RUN apt-get install -y libhdf5-dev
 
-# Configure apt and install packages
-RUN apt-get update \
-    && apt-get -y install --no-install-recommends apt-utils git tzdata dialog 2>&1 \
```

Comment:
Do you know if those packages are not actually needed anymore? Especially `tzdata` seems to have been added on purpose to have the timezone tests working: https://github.com/pandas-dev/pandas/pull/46219

(we copied this for the gitpod docker file as well, but so if it's no longer needed, we can clean that up there as well)

Reply:
tzdata is already installed on the image. I'm guessing either from the base python image or may be included as part of the base ubuntu